test/middleware/busted_tracer.rb

```diff
@@ -0,0 +1,26 @@
+require "logger"
```

Comment:
Thanks! I totally missed that I had the file twice.
That was some bad git hygiene on my part.